devenv/local/docker-compose/docker-compose.yml

```diff
@@ -460,74 +460,38 @@ services:
 
   # Modifies the CDK template and creates DynamoDB Tables if necessary.
   aws-setup:
```

Comment:
Should this still be called `aws-setup`, or should it be `emily-setup` or something more descriptive?

Reply:
I'll change it to `emily-setup`